TweetMate/src/main/java/com/ybsystem/tweetmate/activities/ActivityBase.java

```diff
@@ -22,7 +34,8 @@ protected void onCreate(Bundle savedInstanceState) {
         setTheme();
         setBackButton();
 
-        // Init resource
+        // Init resources
+        ResString.init();
         ResColor.init();
     }
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "SearchActivity" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1192)